doc/btp_spec.txt

```diff
@@ -562,6 +562,24 @@ Events:
 		This event can be sent when a connection is established or
 		when the connection parameters have changed.
 
+	Opcode 0x89 - Security Level event
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
+					Security Level (1 octet)
+
+		Possible values for the Security Level parameter map directly
+		to those defined in Core Specification in LE security mode 1:
+					1 = No security (No authentication and no encryption)
```

Comment:
I would not stick into Core Specification in this case. Receiving this event with "No security" level looks odd. Meaning this should be sent after each GAP Connected event. It would be better to send this event when the level changes only IMO.